frontend/src/components/Seats/actions.ts

```diff
@@ -20,26 +21,23 @@ const seatDeleted = (success: boolean) => ({
   type: DELETE_SEAT,
 });
 
-export const createSeatAction = (seat: ISeat):
+export const createSeatAction = (seat: IPostSeat):
   ThunkAction<Promise<void>, {}, {}, AnyAction> => async (dispatch: Dispatch) => {
     const result = await postSeat(seat);
     if (result) dispatch(seatCreated(true, result));
     else dispatch(seatCreated(false));
   };
 
-export const deleteSeatAction = (roomId: number, seatId: string):
+export const deleteSeatAction = (seatId: number):
   ThunkAction<Promise<void>, {}, {}, AnyAction> => async (dispatch: Dispatch) => {
-    const result = await deleteSeat(roomId, seatId);
+    const result = await deleteSeat(seatId);
     if (result) dispatch(seatDeleted(true));
     else dispatch(seatDeleted(false));
   };
 
-export const updateSeatAction = (roomId: number, oldSeatId: string, newSeat: ISeat):
+export const updateSeatAction = (seatId: number, newName: string):
   ThunkAction<Promise<void>, {}, {}, AnyAction> => async (dispatch: Dispatch) => {
-    const deleteResults = await deleteSeat(roomId, oldSeatId);
-    if (deleteResults) {
-      const createResult = await postSeat(newSeat);
-      if (createResult) dispatch(seatUpdated(true));
```

Comment:
On `seatUpdated`, would it perhaps be an idea to update seats in `state.rooms.rooms` as well, so store reflects backend?

Reply:
I've created an issue for this and we'll deal with it after merging this PR. #304